eng/versioning/version_client.txt

```diff
@@ -51,6 +51,7 @@ com.azure:azure-analytics-synapse-artifacts;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-analytics-synapse-spark;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-analytics-synapse-managedprivateendpoints;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-analytics-synapse-monitoring;1.0.0-beta.3;1.0.0-beta.4
+com.azure:azure-communication-callingserver;1.0.0;1.0.0-beta.1
```

Comment:
According to the instructions at the top of the file, it should be `com.azure:azure-communication-callingserver;1.0.0-beta.1;1.0.0-beta.1` right?

Reply:
Modified to 
com.azure:azure-communication-callingserver;1.0.0-beta.1;1.0.0-beta.1